app.js

```diff
@@ -180,34 +184,55 @@
         }
     });
 
-    app.post("/login", function(req, res) {
+app.post("/login", async function (req, res, next) {
+    try {
+
+
```

Comment:
correct the indentation and remove un-necessary whitespaces like these empty lines.

Reply:
@rehanmm Here I meant: give one tab indentation in front of app.post(...) so that indentation scheme matches with the rest of code.